app/models/user.rb

```diff
@@ -1,5 +1,15 @@
 class User < ActiveRecord::Base
   has_secure_password
+  has_many :clients_coaches, class_name: 'CoachClient',
```

Comment:
(minor) I'd change it to `client_coaches` (to be consistent with `coach_clients`)